lib/config/admin.ts

```diff
@@ -3,7 +3,7 @@ import type { RenovateConfig, RepoAdminConfig } from './types';
 let adminConfig: RepoAdminConfig = {};
 
 // TODO: once admin config work is complete, add a test to make sure this list includes all options with admin=true (#9603)
-export const repoAdminOptions = [
+export const repoAdminOptions: (keyof RepoAdminConfig)[] = [
```

Comment:
This causes lint issues

Reply:
Fixed it. Not sure why I can't do `adminConfig[option] = ...` when option is of the right type, but I'm guessing it's because of some type insecurity restriction.